src/configcontainer.cpp

```diff
@@ -168,7 +168,7 @@ ConfigContainer::ConfigContainer()
 		ConfigData("no", ConfigDataType::BOOL)},
 	{
 		"podlist-format",
-		ConfigData( "%4i [%6dMB/%6tMB] [%5p %%] [%7k kb/s] %-20S %u -> %F", ConfigDataType::STR)},
+		ConfigData( "%4i [%6dMB/%6tMB] [%5p %%] [%12k] %-20S %u -> %F", ConfigDataType::STR)},
```

Comment:
Regarding `%12k`:
`xxx.xx xB/s` is only 11 characters, allowing for a 1 char longer translation for KB/s (e.g. if some translator uses `KiB/s` or `MiB/s`).

If longer translations are added, the end will be cut of when printing.
Not sure if it is worth fixing such a potential problem now.

Reply:
I think it can also be 12 characters, e.g. "1023.99 xB/s". Can't in?

> Not sure if it is worth fixing such a potential problem now.

I wouldn't. The problem doesn't seem too dire: translators can always use the English terms, which might look a bit lame in between translated text, but will be understood by the user all right.

What I *would* fix is the alignment: `%12k` pads on the right, so if I have two speeds, "123.45 KB/s" and "1.05 GB/s", they won't line up nicely:

```
... |123.45 KB/s | ...
... |1.05 GB/s   | ...
```

I'd use `%-12k` there, for left padding:

```
... | 123.45 KB/s| ...
... |   1.05 GB/s| ...
```

Reply:
> What I would fix is the alignment: `%12k` pads on the right

The documentation tells the same thing

> positive numbers indicate that the text that is inserted for the sequence shall be padded right to a total width that is specified by the number, while negative number specify left padding

but it looks like it is working exactly the other way around (at least in this case).
By default it is right aligned and when I apply your suggestion (`12k`->`-12k`) al text is printed on the left which results in ugly misalignment.

- [ ] I will look into this



> I think it can also be 12 characters, e.g. "1023.99 xB/s". Can't in?

You're right, I guess that is possible.

Reply:
It looks like the documentation has left/right inverted with respect to padding (or padding/alignment was mixed up).

I've created a pull request to fix the documentation: https://github.com/newsboat/newsboat/pull/745
I expect that resolves this part (if I'm not mistaken regarding the padding).